pkg/spec/backup.go

```diff
@@ -73,6 +73,9 @@ type BackupServiceStatus struct {
 
 	// BackupSize is the total size of existing backups in MB.
 	BackupSize float64 `json:"backupSize"`
+
+	// StorageStatus is the status of the storage the backup pod uses.
+	StorageStatus *StorageStatus `json:"storageStatus,omitempty"`
```

Comment:
There are too many layer of status:
```
status:
  backupStatus:
    storageStatus:
      pvStatus:
```

This is good ehough:
```
status:
  backupStatus:
      pv:
        name:
```

If you agree, then I would suggest make this `inline`, and change the names too.